src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -10,5 +10,7 @@
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The contact index provided is invalid";
     public static final String MESSAGE_INVALID_TYPE = "The type provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
-
+    public static final String MESSAGE_INVALID_LESSON_DISPLAYED_INDEX = "The lesson index provided is invalid";
+    public static final String MESSAGE_MODULE_DOES_NOT_EXIST = "There is no module matched the given module code";
```

Comment:
Should be `There is no module matched to the given module code`. Missing the `to`.

Reply:
Do you feel "There is no module that matches the given module code" is better?

Reply:
Yes this sounds better